src/controller/users.js

```diff
@@ -0,0 +1,29 @@
+const { Router } = require('express');
+const users = require('../data/users');
+
+const router = Router();
+
+
+// eslint-disable-next-line consistent-return
+async function loginUser(req, res, next) {
+  const { userName, passWord } = req.body;
+  if (!userName || !passWord) {
+    return res.sendStatus(400);
+  }
+  try {
+    const [user] = users;
+    if (user.userName === userName && user.passWord === passWord) {
+      const accessTocken = 'little-pony';
+      const { userId } = user;
+      return res.send({ userId, accessTocken });
+    }
+    return res.status(400).send({ message: 'Wrong password' });
+  } catch (err) {
+    next(err);
+  }
```

Comment:
no middleware available

Reply:
we can consider using Passport for authentication. http://www.passportjs.org/
It's also a good idea to implement a simple one, it helps to understand the bais of web-based authentication.